src/Stryker.Core/Stryker.Core/TestRunners/ITestRunner.cs

```diff
@@ -2,7 +2,7 @@
 {
     public interface ITestRunner
     {
-        TestRunResult RunAll();
+        TestRunResult RunAll(int timeoutMS = 0);
```

Comment:
Why does this interface need a default? I think it should be required here and optionally given a default higher up in the chain.